Revamp ITR documentation (#17286)
DataDog/documentation: content/en/continuous_integration/intelligent_test_runner/dotnet.md

---
title: Intelligent Test Runner for .NET
kind: documentation
aliases:
  - /continuous_integration/intelligent_test_runner/dotnet
further_reading:
    - link: "/continuous_integration/tests"
      tag: "Documentation"
      text: "Explore Test Results and Performance"
    - link: "/continuous_integration/troubleshooting/"
      tag: "Documentation"
      text: "Troubleshooting CI"
---

## Compatibility

Intelligent Test Runner is only supported on `dd-trace>= 2.22.0` (execute `dd-trace --version` to get the version of the tool).

## Setup

> Prior to setting up Intelligent Test Runner, you must have finished setting up [Test Visibility for .NET][1]. If you are reporting data through the Datadog Agent you need to use v6.40+/v7.40+.

Reviewer: ```suggestion
Prior to setting up Intelligent Test Runner, set up [Test Visibility for .NET][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
```

> To enable Intelligent Test Runner, the following environment variables need to be set:

Reviewer: ```suggestion
To enable Intelligent Test Runner, set the following environment variables:
```
